src/main/resources/templates/mainView.html

```diff
@@ -2,6 +2,8 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
This two meta tags are not necessary IMO. We are not doing mobile version. Also you can put this link https://litmus.com/community/discussions/219-which-meta-tag-is-the-best-for-responsive on pull request comment for clarification what these two meta tags are for.